conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/Lorre.scala

```diff
@@ -10,9 +16,8 @@ import tech.cryptonomic.conseil.indexer.bitcoin.BitcoinIndexer
 
 import scala.concurrent.Await
 import scala.concurrent.duration._
-import tech.cryptonomic.conseil.common.config.Platforms.EthereumConfiguration
 import tech.cryptonomic.conseil.indexer.ethereum.EthereumIndexer
-import tech.cryptonomic.conseil.common.config.Platforms.QuorumConfiguration
```

Comment:
maybe this is an error?
```
    case conf: QuorumConfiguration =>
      logger.info("Initializing indexer for Quorum Blockchain.")
      EthereumIndexer.fromConfig(lorreConf, conf.toEthereumConfiguration)
```

Reply:
I think its intentional, there isn't another indexer for Quorum and throughout the code there is support for Quorum inside Ethereum classes